requirements_sweeps.txt

```diff
@@ -0,0 +1 @@
+sweeps
```

Comment:
we should pin this to 0.10.0

Reply:
or at least require >=0.10.0. this may break local controller if someone uses e.g. sweeps == 0.1.0

Reply:
sure, np, son't have anything against it -- done. would you mind explaining why we should be specific about it?